src/components/button/button.module.css

```diff
@@ -122,8 +122,6 @@
 }
 .button--interactive_0--bordered:focus {
   outline: 0;
-  box-shadow: inset 0 0 0 var(--border-width-medium)
```

Comment:
Ah, here I see that I have removed too much. The intention was actually to only remove var(--interactive-interactive_0-outline-background), so that the green color with poor contrast against the blue background is replaced with a white color. Will fix👍